### PR TITLE
[DataBoxEdge]Adding cmdlets for the first trime for DataBoxEdge

### PR DESCRIPTION
**Pull request by raviten**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/391) (_Microsoft internal only_)
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/373) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by msJinLei** [comment]
@raviten duplicate with #10274?

**Comment by raviten** [comment]
This is a wrong format of Code, moved to this
https://github.com/Azure/azure-powershell/pull/10279

CMD LET PR review
- https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/391
- https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/373

**Comment by raviten** [comment]
> @raviten duplicate with #10274?

Yes please ignore #10274

**Comment by msJinLei** [comment]
@raviten Could you fix the CI errors?

**Comment by raviten** [comment]
Sure

**Comment by raviten** [comment]
> @raviten Could you fix the CI errors?

Fixed the CI Errors

File: src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/DataBoxEdgeBandwidthScheduleTests.ps1 [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    $dfname = Get-DeviceName
+	$bwname = Get-BandwidthScheduleName
+	$bwRateInMbps = 45
+	$bwStartTime = "11:00:00"
+	$bwStopTime = "13:00:00"
+	$bwDaysOfWeek = "Sunday,Saturday"
+
+    # Test
+	try
+    {
```
Comment (msJinLei) [inline src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/DataBoxEdgeBandwidthScheduleTests.ps1]: indent errpr

Comment (raviten) [reply src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/DataBoxEdgeBandwidthScheduleTests.ps1]: Yes, Fixed Indentation

File: src/DataBoxEdge/DataBoxEdge/Az.DataBoxEdge.psd1 [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+NestedModules = @('.\Microsoft.Azure.PowerShell.Cmdlets.DataBoxEdge.dll')
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'Get-AzDataBoxEdgeJob', 
+				  'Get-AzDataBoxEdgeDevice', 'New-AzDataBoxEdgeDevice', 'Remove-AzDataBoxEdgeDevice',
+				  'Get-AzDataBoxEdgeDeviceExtendedInfo',
```
[the remark this answers is not in the document]
Comment (raviten) [reply src/DataBoxEdge/DataBoxEdge/Az.DataBoxEdge.psd1]: Need to remove it. As It has been moved to Get-AzDataBoxEdgeDevice -ExtendedInfo
Will remove it

**Comment by isra-fel** [comment]
@msJinLei now CI passes. Please review it. Thanks

**Pull request merged**